server/settings/prod.conf

```diff
@@ -0,0 +1,3 @@
+COMPOSE_FILE=docker-compose-prod.yml

```

Comment:
(optional) this can be hardcoded or saved inside dockerStart.py. There's no need for an additional file.